dbt_project.yml

```diff
@@ -10,31 +10,33 @@ macro-paths: ["macros"]
 
 
 on-run-start:
-    - "{{resume_warehouse(var('resume_warehouse', false), var('warehouse_name'))}}"    
+    - "{{resume_warehouse(var('resume_warehouse', false), var('warehouse_name'))}}"
 
 on-run-end: 
     - "{{suspend_warehouse(var('suspend_warehouse', false), var('warehouse_name'))}}"
     - "grant usage on schema {{target.schema}} to role REPORTER"
     - "grant usage on schema {{target.schema}} to role LOADER"
-    
+
 models:
     post-hook: "grant select on {{this}} to role REPORTER"
-    
+
     perfect_keto:
         vars:
-            warehouse_name: 'LOADING'   
-        
+            warehouse_name: 'LOADING'
+
     facebook_ads:
           vars:
```

Comment:
i could be wrong but doesn't this need a `router` specified?

Reply:
nope!